okta_openvpn.py

```diff
@@ -222,8 +215,8 @@ def auth(self):
                     check_count = 0
                     fctr_rslt = 'factorResult'
                     while fctr_rslt in res and res[fctr_rslt] == 'WAITING':
-                        print("Sleeping for {}".format(
-                            self.mfa_push_delay_secs))
+                        print(("Sleeping for {}".format(
+                            self.mfa_push_delay_secs)))
```

Comment:
Is this just adding another layer of parens?  Whyfore?

Reply:
_Probably_ [this](https://github.com/bigcommerce/okta-openvpn/pull/1#discussion_r1698987314).